app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -43,6 +49,13 @@ public class VroV2Tests {
 
   private final RestTemplate restTemplate = new RestTemplate();
 
+  // Authorization for Claim Info e2e since it cannot be accessed unauthorized.
+  private HttpEntity<Void> getAuthorizationHeader() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("X-API-Key", "test-key-01");
+    return new HttpEntity<>(headers);
+  }
```

Comment:
Minor OCD thing: can we move this down to where Dimitri's `private static HttpHeaders getHttpHeaders()` method is (to keep like-minded code together in case we gotta move it somewhere else later) and probably rename his old one to something more descriptive like `getBearerAuthHeaders` while we're there? Looks like he's only using it in one place.

<img width="445" alt="image" src="https://user-images.githubusercontent.com/119633398/223897901-65a65cf3-4d50-49f7-bd62-69bd7e244979.png">